### PR TITLE
fix: add 500 err page and fix i18n format message

### LINKED ISSUE #69: Fix missing translation and invalid format on npm run build
@404answernotfound  @nicpuppa something is going on the i18n at build time. Probably we've forgot to format a i18n variable in our components. Could you check this?

### PR DESCRIPTION
**Pull request by TheJoin95**
Closes #69

### CODE REVIEW

**Comment by Wabri** [comment]
This seems my fault 💯

File: public/locales/es.json [hunk 1]
```diff
@@ -136,7 +145,7 @@
     "length": "24 marzo 2023",
     "type": "Venue",
     "description": "Mantente en contacto a través de nuestros canales (Discord, LinkedIn) para saber qué sede nos acogerá para este evento :)",
-    "heading": "¿Quieres saber más sobre la sede de este evento? :) Mantente informado en LinkedIn, Twitter o mejor aún, <link>Discord</link>!"
+    "heading": "Quiere saber más sobre el lugar :) ¡Manténgase actualizado en <linkedin>LinkedIn</linkedin>, <twitter>Twitter</twitter> o mejor aún, <discord>Discord</discord>!"
```
Comment (Wabri) [inline public/locales/es.json:148]: Is this the cause of everything?

**Review by Wabri — APPROVED** [review]

**Review by davideimola — APPROVED** [review]

File: public/locales/es.json (hunk 1 above)
Comment (TheJoin95) [reply public/locales/es.json:148]: This one was an actual error, in addition of the others "undefined missing translation" message

**Comment by TheJoin95** [comment]
btw not your fault, but our fault. We are a team, broski

**Pull request merged**